cmd/draftd/start.go

```diff
@@ -70,6 +74,7 @@ func newStartCmd(out io.Writer) *cobra.Command {
 	f.StringVar(&sc.registryURL, "registry-url", "127.0.0.1:5000", "the URL of the registry (e.g. quay.io, docker.io, gcr.io)")
 	f.StringVar(&sc.basedomain, "basedomain", "", "the base domain in which a wildcard DNS entry points to an ingress controller")
 	f.StringVar(&sc.tillerURI, "tiller-uri", "tiller-deploy:44134", "the URI used to connect to tiller")
+	f.StringVar(&sc.storageEngine, "storage", "inprocess", "storage engine draftd should use")
```

Comment:
will we change the default from inprocess to configmap in a later release?

Reply:
I have no issue with that. Perhaps, post or with `logs` next cycle?